R/pkg/inst/tests/testthat/test_context.R

```diff
@@ -177,6 +177,13 @@ test_that("add and get file to be downloaded with Spark job on every node", {
   spark.addFile(path)
   download_path <- spark.getSparkFiles(filename)
   expect_equal(readLines(download_path), words)
+
+  # Test spark.getSparkFiles works well on executors.
+  seq <- seq(from = 1, to = 10, length.out = 5)
+  f <- function(seq) { readLines(spark.getSparkFiles(filename)) }
+  results <- spark.lapply(seq, f)
+  for (i in 1:5) { expect_equal(results[[i]], words) }
+
```

Comment:
does it work when you run `run-tests` on your machine? what are the parameter (esp. --master) you use wth spark-submit?

it looks like the UDF function itself has failed
just a thought - perhaps `SparkR` package needs to be loaded within the UDF?